tck/src/main/java/org/apache/jdo/tck/api/jdohelper/IsDetached.java

```diff
@@ -35,29 +36,23 @@ public class IsDetached extends DetachTest {
   private static final String ASSERTION_FAILED =
       "Assertion A8.5.6-1 JDOHelper.isDetached(Object) failed: ";
 
-  /**
```

Comment:
I do not think so that someone wants to call the main method. 
To run a TCK test you have to use the RunTCK plugin, in oder to get everything initialized that is requied by the TCK test classes.